### ISSUE TITLE
Stumpy strong

### ISSUE THREAD

**Issue by Antarac** [techannihilation/TA#1262]
A player complained that hordes of stumpys
(and similar tanks from other factions) are to strong.
I cannot say that its totally wrong.

**Comment by Leggir** [comment]
Agreed, however at an additional cost and lack of ability to go over uneven terrain, these tanks are fairly well balanced.

**Comment by skymyj** [comment]
They have good balanced. No issue here.
But vs Bot, they are just OP. 
I thing the main issue here is the Rockos and Plasma Kbot balance.
